packages/rule-no-broken-links/src/rule.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @fileoverview This rule checks every link and image tag in the page and report if it is broken
+ * Checks for 404, 410, 500 or 503 status
+ */
+
+import { Category } from 'sonarwhal/dist/src/lib/enums/category';
+import { RuleContext } from 'sonarwhal/dist/src/lib/rule-context';
+import {
+    IRule,
+    ElementFound,
+    RuleMetadata,
+    IAsyncHTMLElement
+} from 'sonarwhal/dist/src/lib/types';
+import { debug as d } from 'sonarwhal/dist/src/lib/utils/debug';
+import { RuleScope } from 'sonarwhal/dist/src/lib/enums/rulescope';
+import { Requester } from 'sonarwhal/dist/src/lib/connectors/utils/requester';
+import { URL } from 'url';
+import { NetworkData } from 'sonarwhal/dist/src/lib/types';
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class NoBrokenLinksRule implements IRule {
+    public static readonly meta: RuleMetadata = {
+        docs: {
+            category: Category.accessibility,
```

Comment:
I'm not sure `accessibility` is the right category for this one. Maybe `misc`?  What does @sonarwhal/core think?

Reply:
> `misc`

👍

Reply:
Ddi you mean "other" ? "misc" does not exist!

Reply:
> Ddi you mean "other" ?

[Yes](https://github.com/sonarwhal/sonarwhal/blob/cb07f57e74486a731db9ab54c4bc576df05ef34f/packages/sonarwhal/src/lib/enums/category.ts#L5). Sorry for the confusion.